src/Collect/Support/Carbon.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Tightenco\Collect\Support;
```

Comment:
New file... do we need this? Does it support something that the current version isn't supporting?

Reply:
An Arr test needs it:

![image](https://user-images.githubusercontent.com/3182864/35657474-d1565d92-06e3-11e8-9a0d-a0ce5c9a2b01.png)

Reply:
Kinda weird to pull it in just because we need it for a test... any ideas of easy ways to make this not necessary? It's not the end of the world I guess?

Reply:
I agree. But we would have to delete one test method, but this can be a little bit tricky :)

Reply:
OK. Fair. Simpler to have an unnecessary class than to parse out individual methods.